src/package.ps1

```diff
@@ -304,7 +304,7 @@ function RemovePackage {
 	if ($null -ne $err) {
 		throw $err
 	}
-	Remove-Item ($Pkg | ResolvePackageRefPath) -ErrorAction Ignore
+	try { (Get-Item ($Pkg | ResolvePackageRefPath) -ErrorAction Ignore).Delete() } catch { } # Workaround stupidity with Remove-Item on older PowerShell
```

Comment:
```diff
function RemovePackage {
	param (
		[Parameter(Mandatory, ValueFromPipeline)]
		[Collections.Hashtable]$Pkg
	)
	$db, $digest, $err = $Pkg | UninstallPackage
	if ($null -ne $err) {
		throw $err
	}
-   Remove-Item ($Pkg | ResolvePackageRefPath) -ErrorAction Ignore
	WriteHost "Untagged: $($Pkg.Package):$($pkg.Tag | AsTagString)"
	if ($null -ne $digest) {
		$content = $digest | ResolvePackagePath
		if (Test-Path $content -PathType Container) {
			[IO.Directory]::Delete($content, $true)
		}
		WriteHost "Deleted: $digest"
	}
+	Remove-Item ($Pkg | ResolvePackageRefPath) -ErrorAction Ignore
	$db | OutPwrDB
}
```

This is the only change I think we need to make. Before, it was trying to remove the junction before the content got deleted.

Reply:
That change should be fine for the `RemovePackage` function, but the one in `PullPackage` will still need to change to support package updates to a newer version.

Reply:
Another alternative is to use `cmd /c rd $refdir 2> NUL`. That's what I was using for most of my testing. I changed it to use the more PowerShell-friendly `Remove-Item` just before making the pull request.

Reply:
Maybe we change it to the .NET directory delete call then

Reply:
Which one is that? `System::IO::Directory::Delete()`? It states that the directory must be empty, so I don't think it will work.

Reply:
Based on what I've seen, I'd recommend the `cmd /c rd...` call. It seems to be the only thing capable of reliably removing a directory junction without deleting all the contents of the directory.

Reply:
I opened a new PR for this. Feel free to close this one if you are okay with that one.

Reply:
> Which one is that? `System::IO::Directory::Delete()`? It states that the directory must be empty, so I don't think it will work.

Overload with a boolean. I use it in this function

Reply:
Are you suggesting just for this function or for both functions? In this function, your suggestion of using `Remove-Item` and moving it down is fine with me, if you are okay with having different implementations in the functions.

But I think the only two options for the `PullPackage` function are `try { (Get-Item $refpath -ErrorAction Ignore).Delete() } catch { }` or `cmd /c rd $refpath 2> NUL`. (I just assumed you would want them both to be the same. But if that's not the case, then moving this one down is fine.)